utils/pofix.py

```diff
@@ -83,6 +85,7 @@
 ("displayed in the upper right, (under the default theme)", "displayed in the upper right (under the default theme)"),
 ("However the few implements", "However, the few implements"),
 ("humans can learn to wield it and able to learn", "humans can learn to wield it and are able to learn"),
+("If no second parameter is supplied, toggle bewteen human and AI.", "If no second parameter is supplied, toggle between human and AI."),
```

Comment:
The lines in pofix are intended to be only short snippets, not entire strings (except if the string itself is only a few words).

Reply:
ok, I shortened them.